frontend_tests/node_tests/stream_sort.js

```diff
@@ -87,6 +95,12 @@ with_overrides(function (override) {
     assert.deepEqual(sorted.normal_streams, ['fast tortoise']);
     assert.deepEqual(sorted.dormant_streams, []);
 
+    // Test searching part of hyphenated word
+    sorted = stream_sort.sort_groups("infra");
+    assert.deepEqual(sorted.pinned_streams, []);
+    assert.deepEqual(sorted.normal_streams, ['fhir-infrastructure']);
+    assert.deepEqual(sorted.dormant_streams, []);
+
```

Comment:
Let's add a test for where the user entered `fhir-in` and make sure that still matches `fhir-infrastructure`; we definitely want it to.